### PR TITLE
Adds support for reading NXopt based nexus file

### PR DESCRIPTION
**Pull request by domna**

### CODE REVIEW

File: src/elli/importer/nexus.py [hunk 1]
```diff
@@ -30,29 +54,86 @@ def read_nexus_psi_delta(nxs_filename: str) -> pd.DataFrame:
...
+                [aois, wavelength], names=["Angle of Incidence", "Wavelength"]
+            ),
+            dtype=float,
+        )
+
+        data = np.array(h5file[data_path])
+
+        for i, aoi in enumerate(aois):
+            psi_delta_df.loc[aoi, "Ψ"] = data[indexing(i, 0)]
+            psi_delta_df.loc[aoi, "Δ"] = data[indexing(i, 1)]
```
Comment (MarJMue) [inline src/elli/importer/nexus.py:79]: So we use this as default data structure for now, which i am totally fine with.

But i am not sure if it breaks calc_rho and fitting for multi aio files. But i have no way to test that quickly.

Edit: Ok calc_rho is obviously no problem.

**Review by MarJMue — APPROVED** [review]
LGTM

File: tests/test_nexus/ellips_nx_opt.test.nxs
Comment (MarJMue) [inline tests/test_nexus/ellips_nx_opt.test.nxs:1]: Github should implement h5web to view nexus files :wink: 

I really grown to like it for data inspection. Thanks for the hint :+1:

File: src/elli/importer/nexus.py (hunk 1 above)
Comment (domna) [reply src/elli/importer/nexus.py:79]: yeah calc_rho essentially assumes this structure and so its what were using rn. Anyways, I think we should change this to a general proper data structure eventually but then also w/ support for arbitrary scan axis.

**Pull request merged**